src/main/osd/osd_elements.c

```diff
@@ -1489,6 +1489,14 @@ static void osdElementWarnings(osdElementParms_t *element)
     }
 #endif // USE_RC_SMOOTHING_FILTER
 
+    // Show warning if mah consumed is over the configured limit
+    if (osdWarnGetState(OSD_WARNING_LOW_CAPACITY) && ARMING_FLAG(ARMED) && osdConfig()->cap_alarm > 0 && getMAhDrawn() >= osdConfig()->cap_alarm) {
+        tfp_sprintf(element->buff, "LOW CAPACITY");
```

Comment:
While I think that the warning makes sense, I think that 'LOW CAPACITY' is the wrong message for the warning - 'capacity' is the term used for the total amount of charge a battery can hold, so 'LOW CAPACITY' does not indicate that there is no charge left. How about 'OVER CAP' or simply 'LOW BATTERY'?

Reply:
I've renamed to OVER_CAP, made the most sense to me since the capacity alarm is named cap_alarm.